src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
Using find is generally not wrong, because it returns `undefined` or an object which is a truthy value, but for the sake of writing easier-to-understand code I would replace it with something more "Booleanish". For example a `some` function can be used for that just like in `calculateAverageAgeDiff` function